lib/rspec/openapi/schema_builder.rb

```diff
@@ -48,13 +48,16 @@ def example_enabled?
   def build_parameters(record)
     parameters = []
 
+    marker_to_keep_last_duplicate = Time.now.utc.round.to_s
```

Comment:
The PR is now concise enough to make me understand how and why it's used. Thank you.

I have some suggestions here though:

* Can we insert them inside `SchemaMerger` instead of `SchemaBuilder`? This seems to be the concern for `SchemaMerger` and it feels like the current implementation scatters the implementation details to unrelated places. This PR should touch `SchemaBuilder`.
* Could we achieve the same thing by just always prepending new schema elements to the array when merging them and calling `uniq!` after that? It should give the same result as sorting based on `__marker`.

Reply:
> Can we insert them inside SchemaMerger instead of SchemaBuilder?

It makes sense. https://github.com/k0kubun/rspec-openapi/pull/59/commits/6326392058708f25889524086fb4d82f11f32d16
And it turned out that `__marker` is not needed. https://github.com/k0kubun/rspec-openapi/pull/59/commits/1128a0c0e60b8b12bd06c8c431f4b3d4ca4dd86e

> Could we achieve the same thing by just always prepending new schema elements to the array in SchemaMerger and calling uniq! after that?

Yes, if big diffs on the existing `rails/doc/openapi.(json|yml)` is acceptable.
https://github.com/k0kubun/rspec-openapi/pull/59/commits/2e88f7661a01181dc6994a71110e3c4cde7f379b